ruby/test/integration/minitest_redis_test.rb

```diff
@@ -136,25 +137,27 @@ def test_junit_reporter
            </error>
            </testcase>
          </testsuite>
-         <testsuite name="ATest" filepath="test/dummy_test.rb" skipped="3" failures="1" errors="0" tests="5" assertions="4" time="X.XX">
-           <testcase name="test_bar" lineno="8" classname="ATest" assertions="1" time="X.XX">
+         <testsuite name="ATest" filepath="test/dummy_test.rb" skipped="3" failures="1" errors="0" tests="4" assertions="3" time="X.XX">
+           <testcase name="test_bar" lineno="8" classname="ATest" assertions="1" time="X.XX" flaky_test="false">
            <skipped type="test_bar"/>
            </testcase>
-           <testcase name="test_flaky" lineno="12" classname="ATest" assertions="1" time="X.XX">
-           <skipped type="test_flaky"/>
+           <testcase name=\"test_flaky\" lineno=\"12\" classname=\"ATest\" assertions=\"1\" time=\"X.XX\" flaky_test=\"true\">
```

Comment:
Subtle change here. Because of what I said above `flaky_test` is only `"true"` if it failed. Dunno if that's ok.